staging/src/k8s.io/kubectl/pkg/cmd/autoscale/autoscale.go

```diff
@@ -215,15 +282,22 @@ func (o *AutoscaleOptions) Run() error {
 		mapping := info.ResourceMapping()
 		gvr := mapping.GroupVersionKind.GroupVersion().WithResource(mapping.Resource.Resource)
 		if _, err := o.scaleKindResolver.ScaleForResource(gvr); err != nil {
-			return fmt.Errorf("cannot autoscale a %v: %v", mapping.GroupVersionKind.Kind, err)
+			return fmt.Errorf("cannot autoscale a %v: %w", mapping.GroupVersionKind.Kind, err)
```

Comment:
Nit: 
```suggestion
			return fmt.Errorf("cannot autoscale a %s: %w", mapping.GroupVersionKind.Kind, err)
```
`.Kind` is a string, so it's better to match the print type with that :wink:

Reply:
done